docs/userguide/roleseparation/federated-resource-quota.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Federated ResourceQuota
+---
+
+## Background
+
+With the widespread use of multi-clusters, the administrator may deploy services on multiple clusters and resource management of services under multiple clusters has become a new challenge.
+A traditional approach is that the administrator manually deploys  a `namespace` and `ResourceQuota` under each Kubernetes cluster, where Kubernetes will limit the resources according to the `ResourceQuotas`.
+This approach is a bit cumbersome and not flexible enough.
```

Comment:
Do we have any alternative for `cumbersome`? I'd prefer the one common and simple.
@Tingtal

Reply:
`cumbersome` is acceptable to me. Another way to say it could be:
The disadvantage is that you need to do it one by one, not once for all.

Reply:
Change to `inconvenient`